### ISSUE TITLE
Docs error: incorrect import from `nuxt-prepare/config`

### ISSUE THREAD

**Issue by McPizza0** [johannschopplich/nuxt-prepare#1]
### Environment

- Operating System: Darwin
- Node Version:     v20.2.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  pnpm@8.6.10
- Builder:          vite
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

Install `nuxt-prepare` dependancy
add to `nuxt.config.ts` modules
create `server.prepare.ts`
add line `import { defineNuxtPrepareHandler } from 'nuxt-prepare/config';`

### Describe the bug

Importing from `'nuxt-prepare/config'` fails as `/config` is not a module in itself.
`defineNuxtPrepareHandler` is already re-exported by [module.ts](https://github.com/johannschopplich/nuxt-prepare/blob/f6e2bf05fc66f4c2a069107f9a6e29b73e051b17/src/module.ts#L8) and thus the import can be directly from the package:

`import { defineNuxtPrepareHandler } from 'nuxt-prepare'`

**Comment by johannschopplich** [comment]
Hey there, @McPizza0! Thanks for your contribution. 

Actually, although the import of `defineNuxtPrepareHandler` from `nuxt-prepare` is supported directly, I added the `./config` submodule import path to make [jiti](https://github.com/unjs/jiti) faster. If `defineNuxtPrepareHandler` is imported from `module.ts`, jiti has to load more packages like `@nuxt/kit`, which are referenced in `module.ts`, even if they are not used by `defineNuxtPrepareHandler` itself. 

I believe Nuxt itself just like Vitest provide a `/config` submodule. That's why I chose it. 🙂 

Having said that, it seems like your IDE didn't pick up the typings. This is fixed by https://github.com/johannschopplich/nuxt-prepare/commit/9b938db451fa9a6d2f8708ec3011459587a24021. The fix is available in ✅ [v0.1.2](https://github.com/johannschopplich/nuxt-prepare/releases/tag/v0.1.2).

I wasn't aware of that issue, since I'm using TS bundler module resolution in my projects:

```ts
export default defineNuxtConfig({
  typescript: {
    tsConfig: {
      compilerOptions: {
        moduleResolution: 'bundler',
      },
    },
  },
})
```
